platforms/astro.ts

```diff
@@ -0,0 +1,23 @@
+import type { APIContext, APIRoute } from 'astro'
+
+import type { WorkflowServeOptions, WorkflowContext } from "../src";
+import { serve as serveBase } from "../src";
+
+export function serve<TInitialPayload = unknown>(
+  routeFunction: (
+    apiContext: APIContext,
+    workflowContext: WorkflowContext<TInitialPayload>
```

Comment:
```suggestion
    workflowContext: WorkflowContext<TInitialPayload>,
    apiContext: APIContext
```

Can we change the order of the parameters?